core/src/queue.rs

```diff
@@ -601,42 +647,6 @@ mod tests {
         assert_eq!(queue.get_transactions_for_block(&wsv).len(), 0);
     }
 
-    // Queue should only drop transactions which are already committed or ttl expired.
```

Comment:
I have decided to get rid of this invariant. 
Because transactions are loaded into the sumeragi transaction cache and it's waste of resources to repeatedly bombard transaction cache with transaction duplicates.